oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -513,6 +513,16 @@ public APIGatewayProxyResponseEvent handleRequest(
                                 clientRedirectURI, authCode, null, null, state, null, responseMode);
 
                 sessionService.storeOrUpdateSession(userSession);
+                orchSessionService.updateSession(
```

Comment:
Is there a risk of _"downgrading"_ a user's credential trust on the session here if we set without the [comparison done here?](https://github.com/govuk-one-login/authentication-api/pull/5585/files#diff-e561fad081fa2ac0c63ebf9b039a877a099b78ab42d5a907b5616c74b5395c73R498-R503) It's probably something that rarely really happens but this looks like a user could go from medium -> low

Reply:
This has been resolved in the latest changes